core-bundle/src/Resources/contao/models/PageModel.php

```diff
@@ -862,10 +862,16 @@ public function loadDetails()
 					// Page title
 					if ($type != 'root')
 					{
+						// If $folderUrl is not yet set, use the alias of the first
+						// parent page if it is not a root page (see #2129)
+						if (!$folderUrl)
```

Comment:
```suggestion
						if (!$folderUrl && $objParentPage->alias)
```
Would it make sense use the first parent page that has an alias?

Reply:
Yes, I think so. 👍🏻